Libraries/Image/RCTImageLoader.m

```diff
@@ -400,7 +400,13 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
             return;
           }
 
-          NSURL *redirectURL = [NSURL URLWithString: location];
+          NSURL *redirectURL;
+          if (![location hasPrefix:@"http"]) {
```

Comment:
@satya164 You're right, hadn't thought of that. Do you think checking if it starts with a `/` covers all the cases?